src/lib/FieldType/Mapper/UserAccountFormMapper.php

```diff
@@ -48,6 +48,21 @@ public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, Field
             'label' => /** @Desc("Password must contain at least one non-alphanumeric character") */ 'field_definition.ezuser.require_at_least_one_non_alphanumeric_character',
         ]);
 
+        $fieldDefinitionForm->add('requireNotCompromisedPassword', PasswordConstraintCheckboxType::class, [
+            'property_path' => $validatorPropertyPathPrefix . '[requireNotCompromisedPassword]',
+            'label' => /** @Desc("Password must not be contained in a public breach.") */
+                'field_definition.ezuser.require_not_compromised_password',
+            'help' => /** @Desc("%begin%This uses the API at %link% to securely check breach data.
+                * The password is not transmitted to the API.%end%") */
+                'field_definition.ezuser.require_not_compromised_password_help',
+            'help_translation_parameters' => [
+                '%begin%' => '<div>',
```

Comment:
I don't think parameters should be used to wrap the text in HTML tags. Unfortunately this will be more difficult as you need to override `form_help` twig block: 
```twig
{% block form_help -%}
    {%- if help is not empty -%}
        {%- set help_attr = help_attr|merge({class: (help_attr.class|default('') ~ ' help-text')|trim}) -%}
        <div id="{{ id }}_help"{% with { attr: help_attr } %}{{ block('attributes') }}{% endwith %}>
            {{- block('form_help_content') -}}
        </div>
    {%- endif -%}
{%- endblock form_help %}
```
Maybe @dew326 can help with frontend part.

Reply:
Yes it looks like this should be fixed in CSS, `ibexa-form-help` has display:flex and this causing the problem.

Reply:
@dew326 I thought flex was good, since with the added div we can word-wrap the string and have the second line align with the first, instead of below the little "i"-icon. I don't know how to do that in pure css. Trying some variations...

Reply:
In that case `<div>` is okay but it has to be moved from translations to template. Let me know if you need any guidance/help with that.

Reply:
Great, divs in all teh help texts coming up then.

Reply:
Seems fine in test.